documentation/images/plantuml/agent/flows/ap_metrics_cert_4.7.4.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+->controller: Execute 4.1.1 or 4.1.2
```

Comment:
Test number is 4.7.4

Reply:
Executing test 4.1.1 or 4.1.2 -part of the test 4.7.4